packages/phone-number-privacy/src/match-making/get-contact-matches.ts

```diff
@@ -30,7 +34,15 @@ interface ContactMatch {
 }
 
 function isValidGetContactMatchesInput(requestBody: any): boolean {
-  return hasValidUserPhoneNumberParam(requestBody) && hasValidContractPhoneNumbersParam(requestBody)
+  return (
+    hasValidAccountParam(requestBody) &&
+    hasValidUserPhoneNumberParam(requestBody) &&
+    hasValidContractPhoneNumbersParam(requestBody)
+  )
+}
+
+function hasValidAccountParam(requestBody: any): boolean {
+  return requestBody.account && (requestBody.account as string).startsWith('0x')
```

Comment:
Consider using the utilities in https://github.com/celo-org/celo-monorepo/blob/master/packages/utils/src/address.ts for stuff like this

Reply:
there's no check for leading 0x here -- just 'ensureLeading0x' which modified the input to add a leading 0x if there is none

Reply:
I was thinking you'd use `isValidAddress` instead of just checking that the account param starts with `0x`

Reply:
oh i see -- its exported from ethereumjs-util

kk i'll update in next pr (as the mergify has already pushed this one -- should have waited for the shipit before adding the label...)

Reply:
oh you added the label :P